### PR TITLE
Add docs for fetchQuery

### PR DESCRIPTION
**Pull request by robrichard**

### CODE REVIEW

File: docs/Modern-fetchQuery.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  }
+`;
+
+const variables = {
+  pageID: '110798995619330',
+};
+
+fetchQuery(environment, query, variables)
+  .then(data => {
+    // access the graphql response
```
Comment (alloy) [inline docs/Modern-fetchQuery.md]: Does this contain just the fulfilled data or does this object consist of `{ data, errors }` ?

Comment (robrichard) [reply docs/Modern-fetchQuery.md]: just the fulfilled `data`. As far as I can tell, there's no way to get the `errors` from this API.

Comment (orta) [reply docs/Modern-fetchQuery.md]: Would it throw?

Comment (alloy) [reply docs/Modern-fetchQuery.md]: You can have both fulfilled data and errors, so I doubt that.

Comment (robrichard) [reply docs/Modern-fetchQuery.md]: The promise is rejected only if the GraphQL response returns `data: null`, similar to what's described in https://github.com/facebook/relay/issues/1913.

Comment (alloy) [reply docs/Modern-fetchQuery.md]: Ah right, makes sense.

**Comment by jstejada** [comment]
Awesome, thanks for adding this! Were you able to run the website locally to test it?

**Comment by robrichard** [comment]
Yes - once it was running I had to change the version to actually see the changes. I don’t think it will show up by default until the next release is cut.

**Comment by jstejada** [comment]
cool, thanks! yes, that is correct, this won't show up until we cut a new version with docusaurus or manually update the versioned docs.

**Pull request closed without merging**